LEAF_Nexus/api/controllers/Group/GET.php

```diff
@@ -18,13 +18,21 @@
     return $group->listGroups($args[0], $_GET['offset'], $_GET['quantity']);
 });
 $this->index['GET']->register('group/[digit]/positions', function($args) use ($group) {
-    return $group->listGroupPositions($args[0]);
+	try {
```

Comment:
Might seem a bit weird, but we should allow the `$group->listGroupPositions()` method to handle any exceptions. The method returns an `Exception` if there is an issue, and the response knows how to handle it.